src/entities/monitor-formula.entity.ts

```diff
@@ -72,7 +72,7 @@ export class MonitorFormula extends BaseEntity {
 
   @OneToMany(
     () => DerivedHrlyValue,
-    c => c.formula,
+    c => c.monitorFormula,
```

Comment:
There's a build error saying it doesn't exist on DerivedHrlyValue. I would double check that file

Reply:
Ah, this must be the branch I branched from the wrong branch. I guess it broke when I reverted changes. Thanks for catching this.